### ISSUE TITLE
Progressive slides + autoHeight 'calc'

### ISSUE THREAD

**Issue by veke** [malsup/cycle2#316]
When using progressive slides and if the tallest element is not on the first visible element group, the height calculation seems to fail.

**Comment by albell** [comment]
Right--unfortunately it really can't be any other way. How can the plugin know the height of elements that haven't even been loaded yet, let alone laid out? So you have to choose between these two approaches. If you are going to use progressive slides you need another solution for auto-height, like a fixed aspect ratio.

**Comment by veke** [comment]
True. I think I made my discovery in a hurry :D
